api/catalog/api/docs/audio_docs.py

```diff
@@ -206,3 +208,17 @@ class AudioComplain(MediaComplain):
         "responses": responses,
         "code_examples": code_examples,
     }
+
+
+class AudioThumbnail:
+    desc = f"""
+thumbnail is an API endpoint to retrieve the scaled down and compressed thumbnail
+of the artwork of an audio track or its audio set.
+
+{refer_sample}"""  # noqa
```

Comment:
That's interesting! I removed it and `flake8` doesn't actually complain at all. Maybe these we're added in old code and never removed after refactoring. I'll remove the other occurrences separately to avoid cluttering this PR.

Reply:
Removed `#noqa` in f8a0204. Created #635 for the rest.